# Supplementary Appendix

Shreya Dimri (Department of Evolutionary Biology, Bielefeld University, Bielefeld, Germany)  
Tuba Rizvi (Department of Evolutionary Biology, Bielefeld University, Bielefeld, Germany)  
Júlio Segovia (Department of Evolutionary Biology, Bielefeld University, Bielefeld, Germany)  
Alfredo Sánchez-Tójar (Department of Evolutionary Biology, Bielefeld University, Bielefeld, Germany)

In [ ]:
pacman::p_load(readr)

# Methods (~2200 words)

Registration and Reporting

We pre-registered our predictions, methods and planned analysis prior to data exploration and analysis \[@dimri2024\]. We followed the PRISMA-EcoEvo guidelines \[@odea2021\] for reporting this study (in @figsupp-checklist). All resources associated with this study, including data and code, are available at \[<https://github.com/shreyadimri/Green_Nest_Material>\] (insert Zenodo citation later).

## Systematic Review

### Literature search

In [ ]:
unique_reference_list <- read_csv("data/unique_reference_list.csv")

Rows: 383 Columns: 12
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr (9): label, title, author, journal, volume, number, pages, doi, abstract
dbl (3): year, search_duplicated, n_duplicates

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.

We conducted a systematic search on Web of science Core Collection and Scopus citation database on 07.09.2022 and updated it on 12.08.2024. The search query was designed in three blocks to included terms for predictor/treatment (“green*” OR ”herb*” OR “aromatic*” AND ”nest*”), population of interest (“bird*” OR ”aves” OR ”avian” OR ”ornithol*” OR “passerine*” OR ”passeriform*” OR “songbird*” OR list of all bird genera) and only experimental studies (”experiment*” OR “manipulat\*”) (for complete search string details see @figsupp-string). We searched the title, abstract and keywords in English language only. Our search found 383

#### Literature search

We conducted a systematic search on Web of science Core Collection and Scopus citation database on 07.09.2022

Eligibility criteria

#### Study selection and eligibility criteria

-   Keyword search: where and when..

-   Justification for development of keyword search

-   Findings/Results of search

-   Exclusion criteria

-   How many papers in final meta-analysis and how many effect sizes

-   PRISMA Diagram link

### Data Extraction

-   What did we extract in priority order

-   Wrote to authors for missing info, used methods to deal with missing info

-   Figures extraction

-   Moderators extracted

### Effect size calculation

### Meta-analysis and meta-regressions

-   main effect model

-   meta-regressions for testing hypothesis

-   meta-regressions for exploratory analyses

Senstivity analysis and Publication Bias Test

### Deviations from registration

## Supplementary Appendix

Supplementary materials

| Checklist Item | Sub-item Number | Sub-item | Reported by authors? | Notes |
|----------------|-----------------|----------|----------------------|-------|
|                |                 |          |                      |       |
|                |                 |          |                      |       |
|                |                 |          |                      |       |

The search was conducted twice, the first time on 07.09.2022 and then repeated on 12.08.2024 to ensure it was the most up to date and did not miss any additional articles on the topic that were published meanwhile. For both the searches, we did not use any time limitation i.e. it included all the published literature till that given date. We used **Advance Search** option for both Web of Science Core Collection Database and Scopus Database using the search query given below.